cli/src/factory_ops/unboxing/network/bmc

```diff
@@ -0,0 +1,136 @@
+#truncate -s 0 ${LOG_FILE}
+
+PRVSNR_ROOT="/opt/seagate/cortx/provisioner"
+SALT_OPTS="--no-color --out-file=${LOG_FILE} --out-file-append"
+
+pvt_ip_a=$(get_pillar_data cluster:srvnode-1:network:data_nw:pvt_ip_addr)
+pvt_ip_b=$(get_pillar_data cluster:srvnode-2:network:data_nw:pvt_ip_addr)
+
+remotes=(
+    ${pvt_ip_a}
+    ${pvt_ip_b}
+)
+
+# Set BMC dhcp network configs
+function set_bmc_dhcp_config {
+
+    for remote in ${remotes[@]}; do
+        _linfo "================================================================================"
+        _linfo "Setting BMC Network Information for DHCP on ${remote}"
+        _linfo "================================================================================"
+
+        remote_execute ${remote} "ipmitool lan set 1 ipsrc dhcp"
+        remote_execute ${remote} "ipmitool lan set 1 access on"
+        _linfo "Sleeping for 30 secs for the settings to take effect"
+        remote_execute ${remote} "sleep 30"
+
+        _linfo "********************************************************************************"
+        _linfo "BMC LAN settings have been updated to: "
+        _linfo "********************************************************************************"
+        remote_execute ${remote} "ipmitool lan print 1"
+        _linfo "********************************************************************************"
+    done
+
+    _linfo "================================================================================"
+    _linfo "        "
+
+    proceed_check
+}
+
+
+# Set BMC static network configs
+function set_bmc_static_config {
+
+    _linfo "================================================================================"
+    _linfo "Setting BMC Network Information for Static IP"
+    _linfo "================================================================================"
+    
+    read -p "BMC IP for Server-A: " bmc_srv_1
+    read -p "BMC IP for Server-B: " bmc_srv_2
+    read -p "Gateway IP for BMC interfaces on both nodes: " gw_ip
+    read -p "Netmask for BMC network on both servers [255.255.255.0]: " netmask
+    [[ -z ${netmask} ]] && netmask=255.255.255.0
+
+    while true; do
+        _linfo "********************************************************************************"
+        _linfo "You have provided the following information:"
+        _linfo "    BMC IP for Server-A:                       ${mgmt_ip_1}"
+        _linfo "    BMC IP for Server-B:                       ${mgmt_ip_2}"
+        _linfo "    Gateway IP for BMC interfaces on both nodes:    ${gw_ip}"
+        _linfo "    Netmask for BMC network on both servers:        ${netmask}"
+        _linfo "********************************************************************************"
+        _linfo "    "
+        read -n1 -p "Given the above information,
+          do you wish to proceed with Static IP based NW configuration
+          for BMC interfaces?(y/n): " _ans
+
+        case $_ans in
```

Comment:
I guess `proceed_check` function was created to reduce copy-paste

Reply:
Yes, missed it here :)